src/Support/Message.php

```diff
@@ -49,7 +49,7 @@ public function to($address, $name = null, $override = false)
         return $this;
     }
 
-    public function cc($address, $name = null)
+    public function cc($address, $name = null, $override = false)
```

Comment:
Cool, thanks.
If override == true, I think we should reset the array (see function to() row 39) to keep the class consistent with the real implementation.